assets/src/edit-story/components/canvas/singleSelectionMovable.js

```diff
@@ -161,6 +161,10 @@ function SingleSelectionMovable({ selectedElement, targetEl, pushEvent }) {
       resizable={actionsEnabled && !isDragging}
       rotatable={actionsEnabled && !isDragging}
       onDrag={({ target, beforeTranslate, clientX, clientY }) => {
+        if (!isDragging) {
```

Comment:
Curious, why didn't setting `onDragStart` work?

Reply:
It does but `onDragStart` is called when the element is clicked without dragging which causes the dnd UI to preemptively show up (see #635), it's more like an `onMouseDown` rather than `onMouseUp`